simple-calculator-mcp/README.md

```diff
@@ -39,15 +41,52 @@ def square(a: int) -> int:
     """Square a number"""
     return a * a
 
+# Health endpoint for monitoring
+@mcp.custom_route("/health", methods=["GET"])
+async def health_check(request: Request) -> JSONResponse:
+    return JSONResponse({"status": "OK"})
 
 if __name__ == "__main__":
-    mcp.run(transport="streamable-http", host="0.0.0.0", port=8000, path="/")
+    mcp.run(transport="streamable-http", host="0.0.0.0", port=8000, path="/mcp/")
```

Comment:
`/mcp/` is already the default path for streamable-http
https://gofastmcp.com/integrations/starlette#basic-usage